tripal_chado/src/TripalStorage/ChadoRecords.php

```diff
@@ -126,16 +126,6 @@ protected function initTable($elements) : bool {
     $table_alias = $elements['table_alias'];
     $delta = $elements['delta'];
 
-    if ($base_table == $chado_table) {
-      if ($base_table != $table_alias) {
-        throw new \Exception(t('ChadoRecords::initTable(). Invalid attempt to '
-            . 'initalize a table as the base table cannot have an alias. '
-            . 'The element tries to set a table alias, which is not supported: '
-            . '@elements',
-            ['@elements' => print_r($elements, TRUE)]));
-      }
-    }
-
```

Comment:
This code had to come out because if the base table is used again later for another join (such as with the `featureloc` table) then it will violate this check.